openfpga_flow/benchmarks/quicklogic_tests/i2c_master_top/i2c_master_top_yosys.blif

```diff
@@ -0,0 +1,2085 @@
+# Generated by Yosys 0.9+2406 (git sha1 470f9532, gcc 9.3.0 -fPIC -Os)
```

Comment:
@lnsharma  I see some benchmarks have .blif files pushed. Are they required or they can be reproduced by yosys?

Reply:
I removed the blif files as well as and2 test case.